web/src/components/atoms/Icon/index.tsx

```diff
@@ -1,6 +1,91 @@
-import { HomeOutlined, DownloadOutlined } from "@ant-design/icons";
+import styled from "@emotion/styled";
```

Comment:
Is there any reason at the moment you made this component to be useable with more than just the antd icons?
Do we have icons we are using that aren't antd that we want to support? Or are you just keeping this as close or the same to Re:Earth as possible?

Reply:
@KaWaite , yes, to have a generic component, where we can use multiple icons formats (to keep components as generic as possible and to have a way to replace antd icons)